cpp/src/io/csv/csv_reader.cu

```diff
@@ -1587,6 +1587,20 @@ gdf_error launch_dataTypeDetection(raw_csv_t *raw_csv,
   return GDF_SUCCESS;
 }
 
+/**
+* @brief Returns true is the input character is a valid digit.
+* Supports both decimal and hexadecimal digits (uppercase and lowercase).
+*/
+__device__
+bool isDigit(char c, bool is_hex){
```

Comment:
This feels like a function that might want to be inlined. Is this not the case?

Reply:
I assume that nvcc would automatically inline the function if it determines that it is the optimal thing to do (based on what I know about C++ compilation). 
I can inline it explicitly, but thought that it's better to leave the decision to the compiler given that I did not benchmark the kernel with/without inlining.